.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,15 @@
-**Replace this placeholder text with a summary of the changes in your PR.
-The more detailed you are, the better.**
+**Replace this placeholder text with a summary of the changes in your PR.**
 
 -----------------
 
-Before submitting a PR make sure the following things have been done (and denote this
-by checking the relevant checkboxes):
+Before submitting a PR mark the checkboxes for the items you've done (if you
+think a checkbox does not apply, then leave it unchecked):
 
-- [ ] The commits are consistent with our [contribution guidelines][1]
-- [ ] You've added tests (if possible) to cover your change(s). Indentation & font-lock tests are extremely important!
-- [ ] All tests are passing (`make test`)
-- [ ] The new code is not generating bytecode or `M-x checkdoc` warnings
+- [ ] The commits are consistent with our [contribution guidelines][1].
+- [ ] You've added tests (if possible) to cover your change(s). Bugfix, indentation, and font-lock tests are extremely important!
+- [ ] Run `M-x checkdoc` and fix any warnings in the code you've written.
 - [ ] You've updated the changelog (if adding/changing user-visible functionality)
```

Comment:
You didn't change the punctuation here as you did with the previous items. The punctuation should be consistent.